styleguide/components/21-atoms/tag/_tag.scss

```diff
@@ -1,28 +1,9 @@
 .tag {
-  @include theme('background', 'color-secondary', 'tag-background');
-  @include theme('color', 'color-primary--lighten-1', 'tag-color');
+  @include theme('background', 'color-primary--lighten-4', 'tag-background');
+  @include theme('color', 'color-tertiary', 'tag-color');
 
-  // @at-root a#{&} {
-  // @include element-states($tag-background, $tag-background-hover, $property: "background");
-  // @include element-states($tag-color, $tag-color-hover, $property: "color");
-  // }
-
-  padding: .05rem .31rem;
-  border-radius: .25rem;
-  font-weight: 700;
-  text-transform: uppercase;
-
-  @at-root a#{&} {
-    text-decoration: none;
-
-    &:active,
-    &:visited {
-      @content;
-    }
-
-    &:hover,
-    &:focus {
-      cursor: pointer;
-    }
-  }
+  height: 1.5rem;
```

Comment:
Is this needed?
Won't it break if your tag wraps over two lines?

Suggest using line-height.

Reply:
Also: inline elements don't have a height property.

Reply:
Fixed it.